app/step-functions-templates/auto_controller_sfn_template.asl.json

```diff
@@ -0,0 +1,57 @@
+{
+  "StartAt": "Receive Event",
+  "States": {
+    "Receive Event": {
+      "Type": "Pass",
+      "Next": "Job Planner"
+    },
+    "Job Planner": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "Output": "{% $states.result.Payload %}",
```

Comment:
Also here, if we're explicit, it makes debugging easier as we can know what to expect as outputs. From the map state inputs below, I assume that we expect an output called 'plans', if so we should do something like so:

```
"Output": {
  "plans": "{% $states.result.Payload.plans %}
}
```

Also note, that with the JSONata SFN configuration, Output can be a little bit precarious, if we were to add in another step inbetween 'Job Planner' and 'For each Job', the states.input would change, it may be best if 'plans' were assigned as a variable instead, and then can be used throughout the rest of the steps